src/components/__test__/NavBar.test.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import {fireEvent, render, screen} from'@testing-library/react';
+import NavBar from '../NavBar';
+import {BrowserRouter} from 'react-router-dom';
+import click from '@testing-library/jest-dom';
+
+describe("<NavBar />", () => {
+    it('render NavBar page',()=>{
+        render(<NavBar />, {wrapper: BrowserRouter})
+    })
+
+    it('Button Click', async ()=>{
+        render(<NavBar />, {wrapper: BrowserRouter})
+        const button =  screen.queryByTestId('manualflow-button')
+        fireEvent.click(button)
+    })
+
+    // it('Show Popup Button Button Click', async ()=>{
```

Comment:
Don't raise MR with commented lines. Fix the test cases or remove it.

Reply:
noted and addressed